include/oneapi/dpl/internal/random_impl/linear_congruential_engine.h

```diff
@@ -141,25 +147,54 @@ class linear_congruential_engine
             state_[__i] = mod_scalar(state_[__i - 1u]);
     }
 
+    // Internal function for calculate degrees of multiplier
+    scalar_type
+    pow_mult_n(unsigned long long __num_to_skip)
+    {
+        ::std::uint64_t __a2;
```

Comment:
Should we change type of _a2 and _num_to_skip to uint64_t or unsigned long long just for better look?

Reply:
It would be not good as unsigned long long is a part of APIs and ::std::uint64_t is implementation